frontend/src/onboarding/NewRepresentationArea.tsx

```diff
@@ -121,8 +122,8 @@ export const NewRepresentationArea = ({
     return <AreaContainer title={title} subtitle="You need edit access to create representations" />;
   } else {
     let subtitle =
-      selection && representationDescriptions.length > 0
-        ? 'Select the representation to create on ' + selection.label
+      selectedItem && representationDescriptions.length > 0
+        ? 'Select the representation to create on ' + selection.entries[0].label
```

Comment:
`selectedItem.label` to stay coherent but it works

Reply:
Done